### ISSUE TITLE
Export email log to csv

### ISSUE THREAD

**Issue by Raikia** [Raikia/FiercePhish#38]

**Comment by nyxgeek** [comment]
Here's a MySQL statement I use to export a list of emails from the db:

`select emails.subject, emails.sent_time, emails.uuid, target_users.first_name, target_users.last_name, target_users.email FROM emails INNER JOIN target_users ON  emails.target_user_id=target_users.id
INTO OUTFILE '/var/lib/mysql-files/Phish_Users.csv'
FIELDS TERMINATED BY ','
ENCLOSED BY '"'
LINES TERMINATED BY '\n';
`

**Comment by Raikia** [comment]
Yeah, in v1.3.0, I'm going to do a pass through everything and add "Export to CSV" for all the different types of data stored
